src/amo/components/AddonReview.js

```diff
@@ -83,25 +83,38 @@ export class AddonReviewBase extends React.Component {
       );
     }
 
+    let placeholder;
+    let prompt;
+    if (review.rating && (review.rating > 3)) {
```

Comment:
Yeah, sorry. My brain got jumbled I meant:

```js
if (review.rating && review.rating > 3) { 
```
would work and I think is easier to read. I prefer the parens for more complicated statements inside `if`s.